docker/examples/elastic/docker-compose.yml

```diff
@@ -60,8 +60,8 @@ services:
     container_name: elastic
 
     ports:
-      - 9300:9300
```

Comment:
@krishnaglodha : this is not closing the port. It is assigning the port to a random port in the host.